twitter_like_app/src/pages/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,27 @@
+import React,{Component} from "react";
+import { Link } from "react-router-dom";
+import { AppBar } from "@material-ui/core";
+import { Toolbar } from "@material-ui/core";
+import { Button } from "@material-ui/core";
+
+
+
+
+
+class Navbar extends Component{
+
+    render(){
+        return(
+                    <AppBar  position='static' color='default'>
+                        <Toolbar>
+                            <Button color="inherit" component={Link} to="/">Home</Button>
+                            <Button color="inherit" component={Link} to="/login">Login</Button>
+                            <Button color="inherit" component={Link} to="/register">Register</Button>
+                            <Button color="inherit" component={Link} to="/profile_page">Profile_Page</Button>
+                        </Toolbar>
```

Comment:
Why you use so strange code formatting? This should look fine 
![image](https://user-images.githubusercontent.com/58670340/136180432-aec3192a-e266-4315-a50f-1fc63bd33817.png)